akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -23,6 +23,10 @@ trait ExceptionHandler extends ExceptionHandler.PF {
 
 object ExceptionHandler {
   type PF = PartialFunction[Throwable, Route]
+  private[http] lazy val errorMessageTemplate: String = {
```

Comment:
Actually, thanks for noticing this - I don't see a reason for that. I guess that `lazy` just slipped there. Also, in your comment above, WDYM by "plus an informative message"? Is it "add an informative message to logs" or "mention ExceptionHandler's informative message in docs"? If it's latter - yes, sure; if the former - do you have any suggestions about what to add to the current message?